android/assets/jsons/translations/template.properties

```diff
@@ -262,6 +262,8 @@ Personality =
 Influence = 
 
 Ally: [civilization] with [amount] Influence = 
+Ally: Unknown civilization with [$allyInfluence] Influence = 
```

Comment:
This is unnecessary since we can use the above line with "Unknown civilization" in place of the civ name